website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -163,6 +163,22 @@ If no filters have been applied, use a [Tooltip](/components/tooltip) coupled wi
 
 ![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
 
+### Avoiding an empty state
+
+Because filtering is often reductive in nature and driven by a user action, it might not always be possible to avoid an empty state within a data set.
+
+If technically possible, use the `count` property within the Dropdown ListItem to more explicitly call attention to filters that will return zero results.
+
+In this example, `status = pending` does not correspond with any results.
+
+![Count property in ListItem](/assets/patterns/filter-patterns/avoid-empty-state-list-item-count.png)
+
+### Reverting filters
+
+If the filters applied to a data set _do_ result in an empty state or no results being returned, provide the user with an escape route to revert clear or all filters at once or individually.
+
+This can be done by adding a "Clear all" action to the [Application State](/components/application-state) that clears all applied filters at once or by including dismissible [Tags](/components/tag) for each applied filter that can be cleared by the user individually.
```

Comment:
```suggestion
This can be done by adding a "Clear all" action to the [Application State](/components/application-state) or by including dismissible [Tags](/components/tag) for each applied filter.
```